### DISCUSSION TITLE
Make category text on menu divider.

### DISCUSSION THREAD

**Discussion by chioio** [tusen-ai/naive-ui#3353] [Ideas]
@07akioni Could you make menu divider tag with different with menu item tag (div), I want add category text on divider ::before element use `tag:nth-of-type(CATEGORY_INDEX)`, not divider line.😁

**Comment by 07akioni** [comment]
I think we won't add it, you'd better using group.

**Reply by chioio** [reply]
But this way has some problem with sider collapsed, like menu item padding size, item children can not show by hover when sider collapsed, group label can't hide, etc. 😥

**Reply by 07akioni** [reply]
I think divider may have problem with sider collapsed. Only submenu supports it.
